src/openforms/config/tests/test_config_check.py

```diff
@@ -41,35 +41,37 @@ def test_access_permission(self):
             self.assertEqual(response.status_code, 200)
 
     def test_disabled_plugins_are_skipped(self):
-        config = GlobalConfiguration.get_solo()
-        config.plugin_configuration = {
-            "registrations": {
-                "stuf-zds-create-zaak": {"enabled": False},
-                "email": {"enabled": True},
-            },
-            "prefill": {
-                "kvk-kvknumber": {"enabled": False},
-            },
-        }
-        config.save()
-
-        user = StaffUserFactory(user_permissions=["configuration_overview"])
-        self.client.force_login(user)
+        with patch(
```

Comment:
nice - however we can avoid the indentation level by moving the patch call to a test method decorator since the configuration values are fixed anyway:

```py
@patch("openforms.config.models.GlobalConfiguration.get_solo", return_value=GlobalConfiguration(...))
def test_disabled_plugins_are_skipped(self, mock_get_solo):
    ...
```